include/z64save.h

```diff
@@ -1442,16 +1442,17 @@ typedef enum {
     /* 1 */ WEEKEVENTREG_HORSE_RACE_STATE_START,
     /* 2 */ WEEKEVENTREG_HORSE_RACE_STATE_2,
     /* 3 */ WEEKEVENTREG_HORSE_RACE_STATE_3,
-    /* 4 */ WEEKEVENTREG_HORSE_RACE_STATE_4,
-    /* 7 */ WEEKEVENTREG_HORSE_RACE_STATE_MAX = 7
+    /* 4 */ WEEKEVENTREG_HORSE_RACE_STATE_4
 } WeekEventRegHorseRaceState;
 
-#define GET_WEEKEVENTREG_HORSE_RACE_STATE (WEEKEVENTREG(92) & WEEKEVENTREG_HORSE_RACE_STATE_MAX)
+#define WEEKEVENTREG_HORSE_RACE_STATE_MASK 7
```

Comment:
Should you define the mask here in a similar manner to `SUBS_OFFER_MODE_MASK`, or'ing all the states? (and `EVENTINF_DOG_RACE_STATE_MASK `)

Reply:
I could here. However, `EVENTINF_DOG_RACE_STATE_MASK` actually wouldn't work since the enum values only go up to 3, which means the `Mask` would only go up to 3. I'd rather both be just 7 since that is the case.